src/status_im/ui/components/camera.cljs

```diff
@@ -34,4 +35,4 @@
   (reagent/create-element default-camera (clj->js (merge {:inverted true} props))))
 
 (defn get-qr-code-data [code]
-  (.-data code))
+  (string/trim (.-data code)))
```

Comment:
```suggestion
  (when (.-data code) (string/trim (.-data code))))
```

Reply:
I updated in both places to use `when-let`, which carries an assumption (I think) that `nil` is okay to pass on even if `(.-data code)` or `@text-value` was `false`. Is it acceptable?